apps/sla/sla.yml

```diff
@@ -24,13 +24,12 @@
 ---
 name: Major Incident
 description: >
-  SLA policy applied to requested or approved
-  Major Incident or CISA Major Incident
+  SLA policy applied only to approved Major Incident
 conditions:
   affect:
     - is not community
   flaw:
-    - is major incident
+    - major incident state is approved
```

Comment:
So this change is unfortunately 1:1 to the original. The similar policy should apply for the CISA MI and also for requested but not yet approved MI - at least without we have an approval that it should be otherwise as in the old tooling it was this way for and intentionally. The right solution could be to create separate policy for each of the three cases with all the same except this condition.